tests/support/cobs/shared.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import type { RadiclePeer } from "../peerManager";
+
+export async function comment(
+  peer: RadiclePeer,
+  objectId: string,
+  message: string,
+  replyTo: string | undefined,
+  options: { projectFolder: string; timestamp: string },
+): Promise<void> {
+  const commentOptions = ["comment", objectId, "--message", message];
+  if (replyTo) {
+    commentOptions.push("--reply-to", replyTo);
+  }
+  await peer.rad(commentOptions, {
+    cwd: options.projectFolder,
+    env: { RAD_COMMIT_TIME: options.timestamp },
+  });
+}
+
+export async function assign(
+  peer: RadiclePeer,
+  objectId: string,
+  assignee: string,
+  options: { projectFolder: string; timestamp: string },
+): Promise<void> {
+  await peer.rad(["assign", objectId, "--to", `did:key:${assignee}`], {
+    cwd: options.projectFolder,
+    env: { RAD_COMMIT_TIME: options.timestamp },
+  });
+}
+
+export async function tag(
+  peer: RadiclePeer,
+  objectId: string,
+  tag: string,
+  options: { projectFolder: string; timestamp: string },
+): Promise<void> {
+  await peer.rad(["tag", objectId, tag], {
+    cwd: options.projectFolder,
+    env: { RAD_COMMIT_TIME: options.timestamp },
+  });
+}
+
+// Increases the timestamp by daily increments
+export function increaseTime(timestamp: string, increase: number): string {
+  return (parseInt(timestamp) + increase * 86400).toString();
+}
```

Comment:
This way we don't need the comment:
```diff
export function increaseTime(timestamp: string, days: number): string {
  return (parseInt(timestamp) + days * 86400).toString();
}
```

Reply:
Inlined this function in the end in the `createOptions` fn to create the `execa` options object